### PR TITLE
Update README.md

### LINKED ISSUE #2: https://www.notion.so/crottys-tech-solutions/How-to-use-verification-0cbf466dce8941bc8f0a494f171fef8d?pvs=4
If you are trying to move a PR you made to `ethereum/eips` over to `ethereum/ercs`, the steps below will guide you through the process:

1. Fork this repository within GitHub:
    1. Navigate to [ethereum/ERCs](https://github.com/ethereum/ERCs).
    1. Find the "Fork" button in the upper right side of the screen and click it.
    1. Proceed with repository creation.

2. Clone your fork locally:
    1. `git clone https://github.com/{user}/ercs`

3. Enter the newly created `ercs` folder:
    1. `cd ercs`

4. Add your fork of the EIPs repository as a remote:
    1. `git remote add eips https://github.com/{user}/eips`

5. Fetch that remote to download your branches:
    1. `git fetch eips`

6. Now checkout the branch related to your original PR:
    1. `git checkout {branch}`

7. Rebase your branch against the ERCs repo:
    1. `git rebase master`
    2. Please note, you do not need to update any references of EIP to ERC; particularly in the relative links. Even though `[ERC-X](./eip-x.md)` doesn't correctly render in this repository, we're currently building the full `eips.ethereum.org` still in the `ethereum/eips` repository. So until we resolve this in #1, we will continue using this relative linking format.

8. Push your branch to your ERCs fork:
    1. `git push -u origin {branch}`

9. Now you can recreate your pull request:
    1. Navigate to https://github.com/ethereum/ERCs/compare
    3. Select your newly created fork and branch
    4. open a new PR to this repo.

Additional notes:

- Do NOT link to ERCs using their filename. Link to ERCs using `[ERC-XXXX](./eip-XXXX.md)`.

_Originally posted by @lightclient in https://github.com/ethereum/ERCs/issues/8_

_Originally posted by @GloWE3 in https://github.com/GloWE3/Rainbowbeast-Webkit/issues/1_

### PR DESCRIPTION
**Pull request by GloWE3**
🔄✨

### CODE REVIEW

**Review by GloWE3 — COMMENTED** [review]
@RainbowScientist5

File: README.md [hunk 1]
```diff
@@ -1,2 +1,42 @@
-# .github
-RainbowScientist-Playground/.github is a ✨special ✨ repository that you can use to add README.md to your public organization profile, visible to anyone. Make sure it’s public and initialize it with a README in the profile directory to get started.
```
Comment (RainbowScientist5) [inline README.md:2]: image

**Review by RainbowScientist5 — COMMENTED** [review]
image[]()```

```

**Review by RainbowScientist5 — COMMENTED** [review]
[0x9c38Bb77B266854cC9A8F65B9a53c5BcEd1852dC]()

**Pull request merged**